packages/containers/src/List/List.sagas.js

```diff
@@ -0,0 +1,32 @@
+import cmf from '@talend/react-cmf';
+import { put, takeEvery, take } from 'redux-saga/effects';
+import Constants from './List.constant';
+
+export function* onToggleFilter(data) {
+	yield put(cmf.actions.components.mergeState('Container(List)', 'default', {
```

Comment:
Hard coding the `componentId` `'default'` can lead to several issues :
- If we instantiate a List with a specific `componentId`, it will try to update the state of someone else.
    - So the state will just not been updated for all others List components than 'default'.
    - If the 'default' one exists, side effect will happen
    - If no 'default' exists, I think it will try to update a non existing/initialized state which is not possible and log an error.
- We can't instantiate 2 or more List connected components without knowing the fact that we need to override the default saga. It breaks the general rule of thumb that a component should be reusable.